modules/standard/Regexp.chpl

```diff
@@ -465,6 +858,16 @@ record regexp {
       _ddata_free(matches);
     }
   }
+
+  /* Perform the same operation as :proc:`regexp.sub` but return a tuple
+     containing the new string and the number of substitutions made.
```

Comment:
From a documentation stand-point, it would be nice if this method and its description followed the one it references.